docs/reference/commandline/daemon.md

```diff
@@ -898,6 +900,8 @@ This is a full example of the allowed configuration options in the file:
 	"cluster-store": "",
 	"cluster-store-opts": [],
 	"cluster-advertise": "",
+	"max-concurrency-download": 3,
+	"max-concurrency-upload": 5,
```

Comment:
@aaronlehmann @MHBauer I temporarily leave the `--max-download/upload-concurrency` alone but let me know if we could find a short name so that I could update the PR.